src/style-stars--5.css

```diff
@@ -0,0 +1,4 @@
+.stars--5 .stars__star {
+  background-image: url(../stars/yellow_star.png);
+  margin-bottom: 0;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). If you have set `margin-top` on the `.stars__star` class in another file, then setting `margin-bottom` here could lead to margin collapse issues. Ensure that you are only applying margin on one side (either top or bottom) for consistent spacing.